timemachine/cpp/src/kernels/k_neighborlist.cuh

```diff
@@ -167,19 +169,19 @@ void __global__ k_find_blocks_with_ixns(
     const double *__restrict__ box,
     unsigned int *__restrict__ interactionCount, // number of tiles that have interactions
     int *__restrict__ interactingTiles,          // the row block idx of the tile that is interacting
-    unsigned int *__restrict__ interactingAtoms, // the col block of the atoms that are interacting
+    unsigned int *__restrict__ interactingAtoms, // [NR * warp_size] atom indices interacting with each row block
```

Comment:
Is `warp_size` correct here? Initially I thought this should be `tile_size`, but the code [here](https://github.com/proteneer/timemachine/blob/a6a9339d68578ca6fa20e46f421b794a5d709037/timemachine/cpp/src/kernels/k_neighborlist.cuh#L399) indicates the former is correct. Is this a case of accidental substitution?

Reply:
It is a little ambiguous right now. I think tile_size is actually the correct variable. Which would mean it should use the tile_size in the code you referenced.

Reply:
I didn't change the referenced code for now as there are similar cases nearby where I'm unsure of the correct constant to use. I'll come back to this when I have a better understanding of the implementation in `k_neighborlist.cuh`.